<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>450</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>-1</number>
     </property>
     <item row="4" column="4">
      <widget class="QPushButton" name="ChangeSign">
       <property name="font">
        <font>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: Gray;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: LightGray
}</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="AllClear">
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: LightCoral;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: DarkRed

}</string>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="ClearEntry">
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: LightCoral;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: DarkRed

}</string>
       </property>
       <property name="text">
        <string>CE</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="Button3">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: Gray;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: LightGray
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="Subtract">
       <property name="font">
        <font>
         <pointsize>27</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: DarkOrange;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: DimGray

}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="Divide">
       <property name="font">
        <font>
         <pointsize>27</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: DarkOrange;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: DimGray

}</string>
       </property>
       <property name="text">
        <string>รท</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="Button1">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: Gray;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: LightGray
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="Button8">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: Gray;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: LightGray
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="Button7">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: Gray;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: LightGray
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="Button4">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: Gray;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: LightGray
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="Button0">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: Gray;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: LightGray
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="Point">
       <property name="font">
        <font>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: Gray;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: LightGray
}</string>
       </property>
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QPushButton" name="RightParenthesis">
       <property name="font">
        <font>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: DarkOrange;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: DimGray

}</string>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="LeftParenthesis">
       <property name="font">
        <font>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: DarkOrange;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: DimGray

}</string>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="Button6">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: Gray;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: LightGray
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="Button5">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: Gray;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: LightGray
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="Multiply">
       <property name="font">
        <font>
         <pointsize>27</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: DarkOrange;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: DimGray

}</string>
       </property>
       <property name="text">
        <string>ร</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="Button2">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: Gray;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: LightGray
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="Button9">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: Gray;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: LightGray
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="Add">
       <property name="font">
        <font>
         <pointsize>27</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: DarkOrange;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: DimGray

}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="Equals">
       <property name="font">
        <font>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: DarkOrange;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: DimGray

}</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="Backspace">
       <property name="font">
        <font>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px outset blue;
   background-color: LightCoral;
   height:50px;
   width:50px;
   cursor:pointer;
	font-size:35 px;
	color: white;
}

QPushButton::pressed{
	background-color: DarkRed

}</string>
       </property>
       <property name="text">
        <string>โบ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="Display">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
